### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by fulv**
Small typo and grammar fixes.

### CODE REVIEW

**Comment by fulv** [comment]
I doubt this needs a changelog entry.

File: README.rst [hunk 1]
```diff
@@ -30,21 +30,21 @@ Documentation
...
 
-Plone is secure by architecture and offers fine grained permission control over content and actions.
+Security is built into Plone's architecture from the ground up.  Plone offers fine grained permission control over content and actions.
 
 Plone is easy to set up, extremely flexible,
 and provides you with a system for managing web content that is ideal for project groups, communities, web sites, extranets and intranets.
 
 - *Plone is easy to install.*
-  You can install Plone with a a click and run installer, and have a content management system running on your computer in just a few minutes.
+  You can install Plone with a a click-and-run installer, and have a content management system running on your computer in just a few minutes.
```
Comment (jensens) [inline README.rst:47]: Is this still true? The Unified-Installer is not that _click-and-run_ as it was in former times with the graphical installer.

File: README.rst [hunk 2]
```diff
@@ -30,21 +30,21 @@ Documentation
...
+For trainings `comprehensive Plone training material <https://training.plone.org>`_ is available.
 
 
 What is Plone?
 --------------
 
 Plone is a ready-to-run content management system, offering a complete set of features needed by a wide variety of organizations.
 
-Plone is secure by architecture and offers fine grained permission control over content and actions.
+Security is built into Plone's architecture from the ground up.  Plone offers fine grained permission control over content and actions.
```
Comment (jensens) [inline README.rst:41]: I think, now that there are two sentences, each has to be on its own line.

File: README.rst (hunk 1 above)
Comment (fulv) [reply README.rst]: Yeah, well, to be honest we should say *Plone is possible to install*

**Pull request merged**